data-explorer/kusto/concepts/data-soft-delete.md

```diff
@@ -110,3 +111,16 @@ To delete all the records that contain data of a given user:
 ### Output
 
 The output of the command contains information about which extents were replaced.
+
+## Soft delete for materialized views
+
+Soft deleted can be executed also on materialized views, where the same concepts explained above are applied similarly for materialized views.
+
+### Syntax
+
+`.delete` [`async`] `materialized-view` *MaterializedViewName* `records` [`with (`propertyName `=` propertyValue [`,` ...]`)`] `<|` *Predicate*
+
+> [!NOTE]
+> The same restrictions on the *Predicate* mentioned above apply here as well.
+> Soft delete is expected to fail in case a materialization of the view is being executed concurrently (specifically, in case there is an intersection between the extents those two commands are executing on).
```

Comment:
I would rephrase to - "Soft delete might fail in case of conflicts with the [materialization process](link to https://learn.microsoft.com/en-us/azure/data-explorer/kusto/management/materialized-views/materialized-view-overview#how-materialized-views-work) running in the background. Retrying the operation can help in this case. To avoid conflicts, you can [disable the materialized view](link to https://learn.microsoft.com/en-us/azure/data-explorer/kusto/management/materialized-views/materialized-view-enable-disable) before executing soft delete, and re-enable it when the operation completes."